mainwindow.ui

```diff
@@ -481,16 +494,31 @@
              </property>
             </widget>
            </item>
-           <item row="8" column="0">
-            <widget class="QCheckBox" name="optimiseCheckBox">
+           <item row="8" column="1">
+            <widget class="QDoubleSpinBox" name="optimiseDoubleSpinBox">
+             <property name="enabled">
+              <bool>true</bool>
+             </property>
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;--optimise&lt;/span&gt;&lt;/p&gt;&lt;p&gt;optimise output gcode, reducing its output size up to 95%, while accepting a very little loss of precision&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
-             <property name="text">
-              <string>Optimise toolpaths</string>
+             <property name="suffix">
+              <string> in</string>
              </property>
-             <property name="checked">
-              <bool>true</bool>
+             <property name="decimals">
+              <number>5</number>
+             </property>
+             <property name="minimum">
+              <double>0.000010000000000</double>
+             </property>
+             <property name="maximum">
+              <double>1.000000000000000</double>
+             </property>
+             <property name="singleStep">
+              <double>0.000010000000000</double>
+             </property>
+             <property name="value">
+              <double>0.000½00000000000</double>

```

Comment:
The value here is strange. Is that valid?

Reply:
A typo ½ and 1 are right beside each other on my keyboard. It's gone now